src/olympia/stats/tests/test_cron.py

```diff
@@ -68,15 +59,6 @@ def test_no_addon_or_date(self, group_mock):
         call_command('index_stats', addons=None, date=None)
         calls = group_mock.call_args[0][0]
 
-        # There should be 3 updates, but 2 of them have a date close enough
-        # together that they'll be indexed in the same chunk, so we should have
-        # 2 calls.
-        update_counts_calls = [
-            call.args for call in calls
-            if call.task == 'olympia.stats.tasks.index_update_counts'
-        ]
-        assert len(update_counts_calls) == 2
```

Comment:
Things like this will be fully removed in a different patch. I removed it here because tests were failing since I changed the fixtures files to only have download counts.